.github/ISSUE_TEMPLATE/request.yml

```
@@ -2,7 +2,7 @@
 name: Service Request
 description: A service request to the MIRSG team
 title: "📥 [REQUEST]: "
-labels: ["request"]
+labels: ["request"] # yamllint disable-line rule:quoted-strings rule:brackets
```

This would be a simpler approach
```suggestion
labels:
  - request
```